applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -470,46 +657,71 @@ public static ConfigurationAsyncClientBuilder builder() {
      * @return Revisions of the ConfigurationSetting
      */
     public Flux<ConfigurationSetting> listSettingRevisions(SettingSelector selector) {
+        return listSettingRevisions(selector, Context.NONE);
+    }
+
+    /**
+     * Lists chronological/historical representation of {@link ConfigurationSetting} resource(s). Revisions are provided
+     * in descending order from their {@link ConfigurationSetting#lastModified() lastModified} date. Revisions expire
+     * after a period of time. The service maintains change history for up to 7 days.
+     *
+     * If {@code options} is {@code null}, then all the {@link ConfigurationSetting ConfigurationSettings} are fetched
+     * in their current state. Otherwise, the results returned match the parameters given in {@code options}.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <p>Retrieve all revisions of the setting that has the key "prodDBConnection".</p>
+     *
+     * <pre>
+     * client.listSettingRevisions(new SettingSelector().key("prodDBConnection"))
+     *     .subscribe(setting -&gt; System.out.printf("Key: %s, Value: %s", setting.key(), setting.value()));</pre>
+     *
+     * @param selector Optional. Used to filter configuration setting revisions from the service.
+     * @param context Additional context that is passed during the service call.
```

Comment:
Please can you file an issue to improve the Context class documentation to include common examples of how to use it (e.g. so it calls out tracing, cancellation, etc).

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/3706